### DISCUSSION TITLE
May I ask what happened to V4 maps on download.mapsforge.org?

### DISCUSSION THREAD

**Discussion by youca04** [mapsforge/mapsforge#1385] [Q&A] [answered]
Hello,

I would like to know what happened to the v4 downloads on the download site? Are they gone forever now?

Does anyone know of any providers (Geofabrik?) who can produce V4 maps?

Regards,
Carl

**Comment by devemux86** [comment, marked as answer]
We now build maps in the latest v5 format (like [OpenAndroMaps](https://www.openandromaps.org/en)).
(which is already quite old, from [2017](https://github.com/mapsforge/mapsforge/blob/master/docs/Specification-Binary-Map-File.md#version-history))

Mapsforge map providers are mentioned [here](https://github.com/mapsforge/mapsforge/blob/master/docs/Mapsforge-Maps.md).

**Reply by youca04** [reply]
Sorry for a follow up, but would it be possible to get hold of the V4 variants while we arrange to migrate to V5 and/or manage to get an Osmosis instance set up)? Or have they been irretrievably deleted?

**Reply by devemux86** [reply]
Maps v3 and v4 were no longer updating.
Creating maps uses a lot of resources, it's better to use them for the latest version.

All apps should be updated over the years (since 2017) so they can read maps v5.

**Comment by applantation** [comment]
The V4 maps are gone -- as Emux pointed out, resources to maintain them put
the newer maps at risk.

The scripts to create them are available online:
https://github.com/mapsforge/mapsforge-creator.

Apologies for the inconvenience, but the download server was never intended
as a resource to rely on for production use.

Ludwig
